src/server/components/formRunner/helpers.ts

```diff
@@ -43,6 +46,20 @@ export function getNewSessionWebhookData(
   return newSessionData;
 }
 
+export async function initialiseFormRunnerSession(
```

Comment:
Ha, I've done a similar thing in this PR https://github.com/UKForeignOffice/lists/pull/498/files#diff-5ad5163ce7b9f0fb67df88b5b5b69ce6a5e50f967636742771248a03bfbcb57dR14

I guess depending on what does in first there will be some conflicts. Or we can agree or where we want this data directory wise and apply it to both PRs 🤷